<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="MacBook-Pro-de-Caique.local" failures="1" tests="5" name="Command line test" time="2.185" errors="0" timestamp="2024-08-15T16:05:15 BRT">
  <testcase classname="testrunner.AirlinesTests" name="createAirline" time="0.446"/>
  <testcase classname="testrunner.AirlinesTests" name="createAirlineAndVerifyResponse" time="0.469">
    <failure type="io.restassured.path.json.exception.JsonPathException" message="Failed to parse the JSON document">
      <![CDATA[io.restassured.path.json.exception.JsonPathException: Failed to parse the JSON document
at io.restassured.path.json.JsonPath$ExceptionCatcher.invoke(JsonPath.java:1002)
at io.restassured.path.json.JsonPath$4.doParseWith(JsonPath.java:967)
at io.restassured.path.json.JsonPath$JsonParser.parseWith(JsonPath.java:1047)
at io.restassured.path.json.JsonPath.get(JsonPath.java:202)
at io.restassured.path.json.JsonPath.getString(JsonPath.java:352)
at testrunner.AirlinesTests.createAirlineAndVerifyResponse(AirlinesTests.java:116)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at testrunner.TestRunner.main(TestRunner.java:20)
Caused by: groovy.json.JsonException: Lexing failed on line: 1, column: 1, while reading '<', no possible valid JSON value or punctuation could be recognized.
at groovy.json.JsonLexer.nextToken(JsonLexer.java:86)
at groovy.json.JsonLexer$nextToken.call(Unknown Source)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:45)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:130)
at io.restassured.internal.path.json.ConfigurableJsonSlurper.parse(ConfigurableJsonSlurper.groovy:97)
at io.restassured.internal.path.json.ConfigurableJsonSlurper$parse.callCurrent(Unknown Source)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:49)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:171)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)
at io.restassured.internal.path.json.ConfigurableJsonSlurper.parseText(ConfigurableJsonSlurper.groovy:83)
at io.restassured.path.json.JsonPath$4$1.method(JsonPath.java:965)
at io.restassured.path.json.JsonPath$ExceptionCatcher.invoke(JsonPath.java:1000)
... 31 more
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- createAirlineAndVerifyResponse -->
  <testcase classname="testrunner.AirlinesTests" name="createAirlineWithDataFaker" time="0.394"/>
  <testcase classname="testrunner.AirlinesTests" name="createAirlineWithDataFakerAndENUM" time="0.395"/>
  <testcase classname="testrunner.AirlinesTests" name="createAirlineWithDefaultData" time="0.473"/>
</testsuite> <!-- Command line test -->
